lib/DeviceGroup.js

```diff
@@ -414,25 +420,26 @@ class DeviceGroup extends Entity {
                         then(() => this._printDevicesList(devices, UserInteractor.MESSAGES.DG_DEVICES_UNASSIGNED, this.identifierInfo));
                 }
                 else {
-                    UserInteractor.printMessage(UserInteractor.MESSAGES.DG_NO_DEVICES, this.identifierInfo);
+                    UserInteractor.printInfo(UserInteractor.MESSAGES.DG_NO_DEVICES, this.identifierInfo);
                 }
             }).
-            then(() => UserInteractor.printSuccessStatus()).
+            then(() => UserInteractor.printResultWithStatus()).
             catch(error => UserInteractor.processError(error));
     }
 
     _printDevicesList(devices, message, ...messageArgs) {
         return this._helper.makeConcurrentOperations(devices, (device) => device._collectShortListData()).
             then(() => {
-                UserInteractor.printMessage(message, ...messageArgs);
-                UserInteractor.printEntities(devices);
+                UserInteractor.printInfo(message, ...messageArgs);
+                UserInteractor.printJsonInfo(devices.map(entity => entity.displayData));
```

Comment:
`printJson`, `printJsonInfo`, `printJsonDialog` look really confusing. It should be `printObj***` I believe instead

Reply:
Yes, you are right. Fixed